src/dmd/expression.d

```diff
@@ -1834,9 +1834,9 @@ extern (C++) abstract class Expression : RootObject
             loc = e.loc;
 
         if (e.op == TOK.type)
-            error("%s `%s` is a type, not an lvalue", e.type.kind(), e.type.toChars());
+            error("`%s` is a `%s` definition and cannot be modified", e.type.toChars(), e.type.kind());
         else
-            error("`%s` is not an lvalue", e.toChars());
+            error("cannot modify `%s`", e.toChars());
```

Comment:
This is not ok.  Why can't it be modified?

Reply:
Ok reverted this.

Reply:
I think it would be good to say "cannot modify \`%s\` because it is not an lvalue", or something like that.

Reply:
Or to keep with the pattern of the other error messages "\`%s\` cannot is not an lvalue, so it cannot be modified"

Reply:
Oh sorry wrong wording. I meant I changed to:

> "`%s` is not an lvalue and cannot be modified